src/test/java/org/fogbowcloud/manager/core/intercomponent/xmpp/requests/RemoteGetUserQuotaRequestHandlerTest.java

```diff
@@ -0,0 +1,139 @@
+import org.fogbowcloud.manager.core.models.ResourceType;
+import org.fogbowcloud.manager.core.models.quotas.ComputeQuota;
+import org.fogbowcloud.manager.core.models.quotas.Quota;
+import org.fogbowcloud.manager.core.models.quotas.allocation.ComputeAllocation;
+import org.fogbowcloud.manager.core.models.tokens.FederationUser;
+import org.jamppa.component.PacketSender;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.xmpp.packet.IQ;
+
+import com.google.gson.Gson;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ RemoteFacade.class, PacketSenderHolder.class })
+public class RemoteGetUserQuotaRequestHandlerTest {
+	private static final String PROVIDINGMEMBER = "providingmember";
+
+	private static final String EXPECTED_QUOTA = "{\"totalQuota\":" + "{\"vCPU\":1,\"ram\":1,\"instances\":1},"
+			+ "\"usedQuota\":{\"vCPU\":1,\"ram\":1,\"instances\":1},"
+			+ "\"availableQuota\":{\"vCPU\":0,\"ram\":0,\"instances\":0}}";
+
+	private static final String TAG_RESULT_ERRO = "\n<iq type=\"error\" id=\"%s\" from=\"%s\">\n"
+			+ "  <error code=\"500\" type=\"wait\">\n"
+			+ "    <undefined-condition xmlns=\"urn:ietf:params:xml:ns:xmpp-stanzas\"/>\n"
+			+ "    <text xmlns=\"urn:ietf:params:xml:ns:xmpp-stanzas\">Unexpected exceptionjava.lang.Exception</text>\n"
+			+ "  </error>\n" + "</iq>";
+
+	private RemoteGetUserQuotaRequestHandler remoteGetUserQuotaRequestHandler;
+	private RemoteFacade remoteFacade;
+	private PacketSender packetSender;
+
+	@Before
+	public void setUp() {
+		this.remoteGetUserQuotaRequestHandler = new RemoteGetUserQuotaRequestHandler();
+		this.remoteFacade = Mockito.mock(RemoteFacade.class);
+		PowerMockito.mockStatic(RemoteFacade.class);
+
+		BDDMockito.given(RemoteFacade.getInstance()).willReturn(this.remoteFacade);
+		this.packetSender = Mockito.mock(PacketSender.class);
+
+		PowerMockito.mockStatic(PacketSenderHolder.class);
+		BDDMockito.given(PacketSenderHolder.getPacketSender()).willReturn(this.packetSender);
+	}
+
+	// test case: When call the handle method passing an IQ request, it must return
+	// the User Quota from that.
+	@Test
+	public void testWithValidIQ() throws Exception {
+		int vCPU = 1;
+		int ram = 1;
+		int instances = 1;
+
+		ComputeAllocation totalQuota = new ComputeAllocation(vCPU, ram, instances);
+		ComputeAllocation usedQuota = new ComputeAllocation(vCPU, ram, instances);
+
+		Quota exprectedQuota = new ComputeQuota(totalQuota, usedQuota);
+		Mockito.doReturn(exprectedQuota).when(this.remoteFacade).getUserQuota(Mockito.anyString(), Mockito.any(),
+				Mockito.any());
+
+		IQ iq = createIq();
+
+		// exercise
+		IQ result = this.remoteGetUserQuotaRequestHandler.handle(iq);
+
+		// verify
```

Comment:
maybe, encapsulate the 2 lines below in a private method here. Take a look in the other Handler test made by @llvieira e @wassoares, we need to keep them consistent.